EVSL_1.0/SRC/landos.c

```diff
@@ -21,8 +21,8 @@
  *    @param[in] msteps number of Lanczos steps
  *    @param[in] npts number of sample points used for the DOS curve
  *    @param[in] *intv Stores the two intervals of interest \\
- *      intv[0:1] = [lambda_min, lambda_max]\\
- *      intv[2:3] = [a b] = interval where DOS is to be computed
+ *      intv[0:1] = [a b] = interval where DOS is to be computed
+ *      intv[2:3] = [lambda_min, lambda_max]\\
  *
```

Comment:
it is better to stick to the convention, intv[0-1]: the interval of interest, [2-3]: lmin, lmax